# Goal
uint32 dishwasher_id
---
# Result
uint32 total_dishes_cleaned
---
# Feedback
float32 percent_complete